pkgs/development/python-modules/sphinxcontrib-blockdiag/2.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, python
+, mock
+, sphinx-testing
+, sphinx
+, blockdiag
+}:
+
+buildPythonPackage rec {
+  pname = "sphinxcontrib-blockdiag";
+  version = "1.5.5";
```

Comment:
generally the convent of `<num>.nix` follows the major version number of the package, not the interpreter version, this file should be `sphinxcontrib-blockdiag/1.nix`

Reply:
Actually I did use `2.nix` initially because it made it easier to find the expressions used by Python 2, but given certain Python 3.x also need it, it did not make much sense to keep it.

Reply:
That's what the "generally" is for :)

Reply:
> generally the convention of `<num>.nix` follows the major version number of the package, not the interpreter version, this file should be `sphinxcontrib-blockdiag/1.nix`

ACK